src/components/data-management/Header.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Dropdown, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Header = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata, canLaunchAnalysis,
+    setUploadModalVisible, openAnalysisModal,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const projects = useSelector((state) => state.projects);
+  const changeDescription = (description) => {
+    dispatch(updateProject(activeProjectUuid, { description }));
+  };
+  return (
+    <>
+      <Row style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Title level={3}>{activeProject.name}</Title>
```

Comment:
this will error out of `activeProject` is set to false, which seems to be the default value set for it in line 19.